src/main.scss

```diff
@@ -68,3 +68,8 @@ aside ul.nav {
     }
   }
 }
+
+// see https://github.com/go-gitea/theme/issues/48
+img {
+  max-width: 100%;
```

Comment:
This applies to all images, as #48 refers to specifically content perhaps some more specific CSS selectors could be used.

Reply:
Fair enough.